src/oc_erchef/custom.mk

```diff
@@ -9,6 +9,23 @@ REL_HOOK = compile bundle
 
 CT_DIR = common_test
 
+ERLANG_DIALYZER_APPS = asn1 \
+                       compiler \
+                       crypto \
+                       edoc \
+                       eldap \
+                       erts \
+                       inets \
+                       kernel \
+                       mnesia \
+                       public_key \
+                       sasl \
+                       ssl \
+                       stdlib \
+                       syntax_tools \
+                       tools \
+                       xmerl
+
```

Comment:
Depending on how it's set in concrete.mk, you might be able to do: 
ERLANG_DIALYZER_APPS += eldap sasl